File: CilixSolutions/Controlador/UsuarioBL.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using AccesoDatos;
using Modelo;
using System.ComponentModel;

namespace Controlador
{
    public class UsuarioBL
    {
        private UsuarioDA usuarioDA;

        public UsuarioBL()
        {
            usuarioDA = new UsuarioDA();
        }

        public bool registrarUsuario(Usuario u)
        {
            return usuarioDA.registrarUsuario(u);
        }

        public BindingList<Usuario> listaUsuarios()
        {
            return usuarioDA.devolverLista();
        }

        public BindingList<Usuario> buscarUsuarios(string id, string rol, string estado)
        {
            return usuarioDA.buscarUsuario(id,rol,estado);
        }

        public bool modificarUsuario(string contrNuevo, string rolNuevo, string idAntiguo)
        {
            return usuarioDA.modificarUsuario(contrNuevo, rolNuevo,idAntiguo);
        }

    }
}

File: Cilix/src/Controlador/UsuarioBL.java

package Controlador;

import AccesoDatos.UsuarioDA;
import Modelo.RolUsuario;
import Modelo.Usuario;
import java.sql.ResultSet;
import java.util.ArrayList;

public class UsuarioBL {
    UsuarioDA accesoDatosUsuario;
    
    public UsuarioBL(){
        accesoDatosUsuario = new UsuarioDA();
    }
    
    public ArrayList<Usuario> devolverUsuarios(){
        return accesoDatosUsuario.devolverUsuarios();
    }
    
    public ArrayList<RolUsuario> listarRoles(){
        return accesoDatosUsuario.listarRoles();
    }

    public ArrayList<Usuario> devolverUsuarios(String s1, String s2) {
        return accesoDatosUsuario.devolverUsuarios(s1,s2);
    }
   
}
